src/client/components/EditableIdField.jsx

```diff
@@ -46,9 +46,18 @@ const EditableIdField = ({
         name={connectorName}
         inline
       />
-      <Button outline onClick={handleDoneEditName} className="util-button">
-        <FaCheck className="oi-icon ms-2" />
-      </Button>
+      <h6 className="align-items-top mb-3">
```

Comment:
The mb-3 css class is probably the cause of the sub-nav growth in edit mode

Reply:
The sub nav grows in edit mode to give space for the potential Error message below the edit. I kept the space for it and aligned the button after the edit to the same level.

Reply:
Would it be possible to add this space only if the error occurs ? It feels weird to have it appear as soon as we set the edit mode and no error appear.